src/stories/components/404/NotFound404.tsx

```diff
@@ -87,17 +62,18 @@ const NotFound404: React.FC = () => {
 
 export default NotFound404;
 
-const Wrapper = styled.div({
+const Wrapper = styled('div')(({ theme }) => ({
   width: '100%',
   height: '100%',
   marginTop: '132px',
   paddingBottom: '128px',
-  [lightTheme.breakpoints.up('table_834')]: {
+
+  [theme.breakpoints.up('table_834')]: {
```

Comment:
Same than above, this is a legacy breakpoint

Reply:
Done